be/src/vec/exec/scan/olap_scanner.h

```diff
@@ -64,7 +64,7 @@ class OlapScanner : public Scanner {
         int64_t version;
         TabletReader::ReadSource read_source;
         int64_t limit;
-        bool aggregation;
+        bool is_pre_aggregation;
```

Comment:
add  some comment to explain the meaning of this parameter. And you could provide some use cases so that other developer could get the meaning more clearly.